### PR TITLE
chore(deps): resync all package-lock, upgrade major versions

### LINKED ISSUE #580: An in-range update of webpack is breaking the build 🚨
## Version **4.17.3** of **webpack** was just published.

<table>
  <tr>
    <th align=left>
      Branch
    </th>
    <td>
      <a href="https://github.com/webpack/webpack-cli/compare/master...webpack:greenkeeper%2Fwebpack-4.17.3">Build failing 🚨</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Dependency
    </th>
    <td>
      <a target=_blank href=https://github.com/webpack/webpack>webpack</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Current Version
    </td>
    <td>
      4.17.2
    </td>
  </tr>
  <tr>
    <th align=left>
      Type
    </td>
    <td>
      devDependency
    </td>
  </tr>
</table>

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

webpack is a devDependency of this project. It **might not break your production code or affect downstream projects**, but probably breaks your build or test tools, which may **prevent deploying or publishing**.

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push:** The Travis CI build could not complete due to an error ([Details](https://travis-ci.org/webpack/webpack-cli/builds/426773573?utm_source=github_status&utm_medium=notification)).
- ✅ **continuous-integration/appveyor/branch:** AppVeyor build succeeded ([Details](https://ci.appveyor.com/project/ev1stensberg/webpack-cli/build/1570)).
</details>

---

<details>
<summary>Release Notes</summary>
<strong>v4.17.3</strong>

<h1>Bugfixes</h1>
<ul>
<li>Fix exit code when multiple CLIs are installed</li>
<li>No longer recommend installing webpack-command, but still support it when installed</li>
</ul>
</details>

<details>
<summary>Commits</summary>
<p>The new version differs by 7 commits.</p>
<ul>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/ee27d36e7cb0bed3ca32911165bbf01fef993e48"><code>ee27d36</code></a> <code>4.17.3</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/4430524921b3dbcb3961dd3bfdb898291e30d7e3"><code>4430524</code></a> <code>Merge pull request #7966 from webpack/refactor-remove-webpack-command-from-clis</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/b717aadf491a784ae1c72d797a626a6778bd8b53"><code>b717aad</code></a> <code>Show only webpack-cli in the list</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/c5eab6751b0fefa5479591029c10c8d097afbb37"><code>c5eab67</code></a> <code>Merge pull request #8001 from webpack/bugfix/exit-code</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/943aa6b455a742395231216e7e54f449c7072dab"><code>943aa6b</code></a> <code>Fix exit code when multiple CLIs are installed</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/691cc94e0f4851ef9f9d3156d45f023f2dfe3217"><code>691cc94</code></a> <code>Spelling</code></li>
<li><a href="https://urls.greenkeeper.io/webpack/webpack/commit/898462db462b501f6779c4de014b6b7890daaf80"><code>898462d</code></a> <code>refactor: remove <code>webpack-command</code> from CLIs</code></li>
</ul>
<p>See the <a href="https://urls.greenkeeper.io/webpack/webpack/compare/dfe6379052598a00f13bfa4f34ac73856ddf3dd0...ee27d36e7cb0bed3ca32911165bbf01fef993e48">full diff</a></p>
</details>

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### PR DESCRIPTION
**Pull request by dhruvdutt**
**What kind of change does this PR introduce?**
Chore

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**

**Summary**
Resync all package-lock.json files which are breaking greenkeeper builds. #580 
Also, upgraded major versions for many packages.

**Does this PR introduce a breaking change?**
No

**Other information**
Fixes #580

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,48 +95,48 @@
...
     "rimraf": "^2.6.2",
     "schema-utils": "^1.0.0",
-    "tslint": "^5.10.0",
+    "tslint": "^5.11.0",
     "typedoc": "^0.12.0",
     "typedoc-plugin-monorepo": "^0.1.0",
     "typescript": "^2.9.2",
-    "webpack": "^4.8.3",
-    "webpack-dev-server": "^3.1.4"
+    "webpack": "^4.18.0",
```
Comment (ematipico) [inline package.json]: Shouldn't we remove these guys? Webpack and dev server. They should be dev peer dependencies. Well, that's outside of the scope the issue

**Comment by ematipico** [comment]
Could please update the version number? It's old. It should be 3.1.0

**Comment by ematipico** [comment]
Could you rebase please?

**Comment by ev1stensberg** [comment]
refetch master in a new branch and open a new PR. look at review comment, next PR that is automated with greenkeeper will be closed

File: package.json (hunk 1 above)
Comment (ev1stensberg) [reply package.json]: we need it to parse plugins. Needs to be sync with at least every major

**Pull request closed without merging**